src/matrand_model.jl

```diff
@@ -0,0 +1,82 @@
+export mat_rand_model
+
+using Noise
```

Comment:
Do we need using Noise here, because we already mention it in RegularizedProblems.jl ?
If so, we might add using Distributions as well.

Reply:
Good catch, thank you!